cdap-data-fabric-tests/src/test/java/co/cask/cdap/data2/transaction/distributed/TransactionServiceTest.java

```diff
@@ -202,6 +210,12 @@ static TransactionService createTxService(String zkConnectionString, int txServi
                            new ZKClientModule(),
                            new DiscoveryRuntimeModule().getDistributedModules(),
                            new TransactionMetricsModule(),
+                           new AbstractModule() {
+                             @Override
+                             protected void configure() {
+                               bind(NamespaceQueryAdmin.class).to(SimpleNamespaceQueryAdmin.class);
```

Comment:
Generally places where we do Modules.override we don't do for the same binding. Having it extracted in its own AbstractModule will make it much simple to use but if you don't want to do it in this PR then I guess that fine too but we should open another jira since this doesn't seem related to CDAP-6534